spec/requests/alert_definitions_spec.rb

```diff
@@ -224,8 +224,8 @@
     expect(response).to have_http_status(:ok)
     expect_result_resources_to_include_hrefs(
       "resources",
-      [api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile.compressed_id, alert_definitions.first.compressed_id),
-       api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile.compressed_id, alert_definitions.first.compressed_id)]
+      [api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile.id.to_s, alert_definitions.first),
+       api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile.id.to_s, alert_definitions.first)]
```

Comment:
I see some usage of the helpers, with the object itself, and in the second argument here, the id specifically.  can this be called as:   api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile, alert_definitions.first) ?